.editorconfig

```diff
@@ -0,0 +1,16 @@
+# EditorConfig <https://EditorConfig.org>
+root = true
+
+# elementary defaults
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = tab
+indent_style = space
+insert_final_newline = false
+max_line_length = 80
```

Comment:
Myslím, že enforceuješ 79

Reply:
To záleží na linteru - resp. na tom co na tom 80tým znaku je.